### ISSUE TITLE
Can't build examples

### ISSUE THREAD

**Issue by khajiit-osx** [boostorg/beast#2149]
First off, thanks so much for a fantastic repo.

### Version of Beast
newest (0.8.2)

### Steps necessary to reproduce the problem

I'm on Ubuntu. When recursively cloning the Boost superproj, running ./bootstrap and then ./b2 headers and then ./b2, I proceed to follow Vinnie's instructions on his readme for Beast, which he says will build the examples (at least the third line will):

```
export PATH=$PWD:$PATH
b2 -j2 libs/beast/test cxxstd=11      # bjam must be in your $PATH
b2 -j2 libs/beast/example cxxstd=11   # "-j2" means use two processors
b2 libs/beast/doc                     # Doxygen and Saxon are required for this
```

When doing this, I get the following output 

```
Performing configuration checks

    - default address-model    : none (cached)
    - default architecture     : none (cached)
    - symlinks supported       : yes (cached)
    - cxx11_constexpr          : no  (cached)
    - cxx11_decltype           : no  (cached)
    - cxx11_hdr_tuple          : no  (cached)
    - cxx11_template_aliases   : no  (cached)
    - cxx11_variadic_templates : no  (cached)
...found 1 target...
```

But no binaries for the examples. I'm not sure what I'm missing?

### All relevant compiler information
g++ on Ubuntu 18.04

**Comment by madmongo1** [comment]
Hmm there’s something odd going on here. The config output suggests that the compiler is not c++11 compatible.

**Comment by khajiit-osx** [comment]
Interesting, I'm working on an AWS workspace and have been compiling with c++11 but all compiling has been abstracted away so I'm not sure exactly how my firm is doing it (I just joined as an intern last week). Is there a way to change the compiler settings used to build with b2?

**Comment by madmongo1** [comment]
Some documentation here
https://boostorg.github.io/build/manual/develop/index.html#bbv2.overview.configuration

I’m not a world expert on b2.

a good place to ask for help is in the #boost channel of cpplang slack

**Comment by madmongo1** [comment]
I guess start by getting b2 to dump debug information. That should provide clues about which compiler it’s found (if any)

**Comment by khajiit-osx** [comment]
Thanks for the advice, after a few hours I finally got it to work (although I honestly have no clue what the fix was because all I did was reclone the boost superproj and follow the same steps as previously mentioned (`./boostrap.sh`, `b2 headers`, and then `b2`). The end of the output from the last command was

```
...updated 1819 targets...


The Boost C++ Libraries were successfully built!

The following directory should be added to compiler include paths:

    /mnt/shares/remotentrn/alt_root/home2/cbarkachi.xtrn/boost

The following directory should be added to linker library paths:

    /mnt/shares/remotentrn/alt_root/home2/cbarkachi.xtrn/boost/stage/lib
```
Silly question, but I still don't see any binaries. When I try to use the cmake files in the beast dir, it's unable to locate anything. I am trying my best to find online resources to address this but I'm not getting very far.
